packages/slate-react/src/components/editor.js

```diff
@@ -121,6 +126,11 @@ class Editor extends React.Component {
       const plugins = this.resolvePlugins(props.plugins, props.schema)
       stack = Stack.create({ plugins })
       schema = Schema.create({ plugins })
+
+      // There may be a better alternative to the 2 lines below.
+      thisCopy.state.stack = stack
+      thisCopy.state.schema = schema
```

Comment:
I believe since everything is by reference that these are the same as acting directly on `this.` instead. So I think the `thisCopy` is redundant. But I'm not sure if it's a good idea or not, or if there's another solution.